mllib/src/test/scala/org/apache/spark/ml/classification/OneVsRestSuite.scala

```diff
@@ -101,6 +101,45 @@ class OneVsRestSuite extends SparkFunSuite with MLlibTestSparkContext with Defau
     assert(expectedMetrics.confusionMatrix ~== ovaMetrics.confusionMatrix absTol 400)
   }
 
+  test("one-vs-rest: tuning parallelism does not change output") {
```

Comment:
So I know this would be annoying to do - but would it make sense to test that we're actually training the models in parallel? I think we're probably doing it correctly right now, but I could see this accidentally getting screwed up and using the wrong execution context in the future.